packages/react-scripts/template/README.md

```diff
@@ -406,7 +406,7 @@ Then in `package.json`, add the following lines to `scripts`:
 ```diff
    "scripts": {
 +    "build-css": "node-sass src/ -o src/",
-+    "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
++    "watch-css": "npm run build-css && node-sass src/*.scss -o src/ --watch --recursive",
```

Comment:
Does this need to be `src/**/*.scss`?
Also, what about `sass` files? Aren't both extensions supported?